vendor/modules.txt

```diff
@@ -516,23 +516,26 @@ k8s.io/test-infra/prow/repoowners
 k8s.io/test-infra/prow/apis/prowjobs/v1
 k8s.io/test-infra/prow/pod-utils/downwardapi
 k8s.io/test-infra/prow/config
+k8s.io/test-infra/prow/errorutil
```

Comment:
@hongkailiu why do these jump around?

Reply:
For me ... always run `make validate-vendor` before pushing PR.
I do not know exactly what triggers the changes on `go.mod` file.

Reply:
OK, lol. Sounds good